mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -789,7 +796,18 @@ const ChatTaskpane: React.FC<IChatTaskpaneProps> = ({
             websocketClient.stream.connect(streamHandler, null);
 
             try {
+                // Check if request was aborted before making the call
+                if (abortController.signal.aborted) {
+                    throw new Error('Request aborted');
+                }
+                
                 const aiResponse = await websocketClient.sendMessage<ICompletionRequest, ICompletionReply>(completionRequest);
+                
+                // Check if request was aborted after receiving response
+                if (abortController.signal.aborted) {
+                    throw new Error('Request aborted');
+                }
+                
```

Comment:
Since these are in the streaming case, we don't need them now. We don't have a stop button in chat mode right now.